packages/api-headless-cms/src/content/plugins/utils/pluralizedTypeName.ts

```diff
@@ -0,0 +1,12 @@
+import pluralize from "pluralize";
+
+// This will make it so if the content model name is a single capitalized letter
+// Ex. A, pluralize will not capitalize the name to AS, and instead make it As
+
+export const pluralizedTypeName = (typeName: string) => {
+    let pluralizedTypeName = pluralize(typeName);
```

Comment:
We mentioned we want this pluralize call written differently. Is there a specific reason why we left it like this? Am I missing something? :)